.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 pkg/*.gem
+Gemfile.lock
```

Comment:
Not having a Gemfile.lock means the code runs on my machine and possibly has mystery bugs on travis or other people whenever a library is updated. By keeping it I have a know good state that I can fall back on whenever I come back to a gem after x years and can just fix bugs instead of fighting the environment.

Also if it is not checked in and you bundle update you can be in a broken state that you have no way of getting out of, with a know good state I then can just back up and update 1 library at a time.